package.json

```diff
@@ -47,37 +47,12 @@
     "@babel/core": "^7.16.7",
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-decorators": "^7.16.7",
-    "@ember/optional-features": "^2.0.0",
```

Comment:
all of the removed dependencies are needed for test-app, not for the addon itself